src/map/graphql.ts

```diff
@@ -0,0 +1,57 @@
+import {
```

Comment:
All the other files to get `search` results are placed in `src/graphql/search`, why not this file?

Reply:
Well, at first everything was seperated but not by feature but by functionality. I moved it here so that it was easy for me to find the `map` module that contains code to plug it into the typeahead and GraphQL modules since they both share the same data retrieval code.